build.gradle

```diff
@@ -526,6 +526,7 @@ project(":samza-hdfs_$scalaVersion") {
       exclude module: 'servlet-api'
       exclude module: 'zookeeper'
     }
+    compile('org.apache.parquet:parquet-hadoop:1.9.0')
```

Comment:
looks like version numbers are all defined in ./gradle/dependency-versions.gradle